docs/lib/content/using-npm/dependency-selectors.md

```diff
@@ -62,6 +62,7 @@ The [`npm query`](/commands/npm-query) command exposes a new dependency selector
 - `:path(<path>)` [glob](https://www.npmjs.com/package/glob) matching based on dependencies path relative to the project
 - `:type(<type>)` [based on currently recognized types](https://github.com/npm/npm-package-arg#result-object)
 - `:outdated(<type>)` when a dependency is outdated
+- `:vuln` when a dependency has a known vulnerability
```

Comment:
could this also accept an optional severity level? iow, i often want to find high vulns but not trivial ones.

Reply:
Eventually

Reply:
ok - why wait?

Reply:
Because it needs proper design.  What about filtering by CWE? What about cvss?  We can't just toss in a new parameter like `:vuln(severity)` because that severely hampers what we can do in the future.  For now a binary `:vuln` gets this initial feature shipped, and doesn't limit what we can do in the future.